code/game/atoms_movable.dm

```diff
@@ -262,6 +260,7 @@
 	return 1
 
 /atom/movable/proc/unlock_atom(var/atom/movable/AM)
+	locking_init()
```

Comment:
Why are you initialising three lists here? You really shouldn't be. You're UNLOCKING an atom, this is your chance to get rid of unused lists, not create them.